bican.tfvars

```
@@ -0,0 +1,6 @@
+singleuser_image_repo = "dandiarchive/dandihub"
+region = "us-east-1"
+singleuser_image_tag = "latest"
+jupyterhub_domain = "hub.bican-kb.com"
+dandi_api_domain = "https://api.dandiarchive.org"
```

Hi @asmacdo, how are you managing users for BICAN?  If from a JSON file, should this variable be left blank?

```suggestion
dandi_api_domain = ""
```

---

My understanding is that there is a JSON file, but it is configuration for the Dandi API server, not the hub directly. If we don't plan to spin up an API server for BICAN, we'll need to add functionality. Or we can leave as is, just using the DANDI user list.

---

I'll defer to Satra, but I would imagine that we would not want to spin up an API server for BICAN.  And wouldn't want all DANDI users to have access to this deployment.

---

ive got an untested implementation here https://github.com/dandi/dandi-hub/pull/158

the problem is that those changes cannot be merged into the main do-eks branch. Any changes to the dandihub.yaml file directly must be kept in a fork.

---

Left this change out-- will address when we merge the rest of https://github.com/brain-bican/kbhub/tree/bican-fork back.